packages/distros/rke2/zarf.yaml

```diff
@@ -0,0 +1,29 @@
+kind: ZarfInitConfig
+metadata:
+  name: "distro-rke2"
+  description: "Used to establish a new Zarf cluster"
+
+components:
+  # AMD-64 version of the rke2 stack
+  - name: rke2
+    import:
+      path: common
+      name: rke2
+    only:
```

Comment:
Since RKE2 only supports AMD64 should we add a dummy component with an after script that returns a non-zero exit code and prints a message that RKE2 does not support ARM (given we have a (somewhat decent) number of ARM users?

Reply:
Might be better to use the architecture key in the package for now as it will prevent building for that architecture and warn as well. E.g. https://github.com/defenseunicorns/zarf/blob/c15f371e26e10bc768522a98c316fed6eb7d1266/packages/big-bang-core/zarf.yaml#L9

Reply:
I've added both, I wanted to keep the placeholder there, and Jeff's change works too